### DISCUSSION TITLE
A problem when parse Description in 'CHARACTERISTIC'

### DISCUSSION THREAD

**Discussion by kdsam2789** [Luncher91/A2LParser#27] [General]
Hi, 

First of all, thank you for developing a good parser.

I need to parse an A2L file, a slight problem when parsing "CHARACTERISTIC".

While parsing a few A2L files, I found a case where "\e" was included in the description as shown below.

`/begin CHARACTERISTIC ASAM_CURVE_STD_AXIS "Curve with \end standard axis" CURVE ...`

ANTLR failed to parse because "\e" is a special character that is not commonly used...

It may be a normal situation to not be able to parse, but I think it is also good to parse A2L even in this case.

When analyzing Description, is it possible to set all the characters between " and " to String object?

**Comment by Luncher91** [comment]
see issue #28
